test/livebook/live_markdown/import_test.exs

```diff
@@ -582,6 +582,112 @@ defmodule Livebook.LiveMarkdown.ImportTest do
+    |> Vl.mark(:line)
+    |> Vl.encode_field(:x, "in", type: :quantitative)
+    |> Vl.encode_field(:y, "out", type: :quantitative)
+    ```
+
+    ```vega-lite
+    {"$schema":"https://vega.github.io/schema/vega-lite/v5.json","data":{"values":[{"in":1,"out":1},{"in":2,"out":2},{"in":3,"out":3},{"in":4,"out":4},{"in":5,"out":5}]},"encoding":{"x":{"field":"in","type":"quantitative"},"y":{"field":"out","type":"quantitative"}},"height":200,"mark":"line","width":500}
+    ```
+    """
+
+    {notebook, []} = Import.notebook_from_markdown(markdown)
+
+    assert %Notebook{
+             name: "My Notebook",
+             sections: [
+               %Notebook.Section{
+                 name: "Section 1",
+                 cells: [
+                   %Cell.Elixir{
+                     source: """
+                     Vl.new(width: 500, height: 200)
+                     |> Vl.data_from_series(in: [1, 2, 3, 4, 5], out: [1, 2, 3, 4, 5])
+                     |> Vl.mark(:line)
+                     |> Vl.encode_field(:x, \"in\", type: :quantitative)
+                     |> Vl.encode_field(:y, \"out\", type: :quantitative)\
+                     """,
+                     outputs: [
+                       vega_lite_static: %{
+                         "$schema" => "https://vega.github.io/schema/vega-lite/v5.json",
+                         "data" => %{
+                           "values" => [
+                             %{"in" => 1, "out" => 1},
+                             %{"in" => 2, "out" => 2},
+                             %{"in" => 3, "out" => 3},
+                             %{"in" => 4, "out" => 4},
+                             %{"in" => 5, "out" => 5}
+                           ]
+                         },
+                         "encoding" => %{
+                           "x" => %{"field" => "in", "type" => "quantitative"},
+                           "y" => %{"field" => "out", "type" => "quantitative"}
+                         },
+                         "height" => 200,
+                         "mark" => "line",
+                         "width" => 500
+                       }
+                     ]
+                   }
+                 ]
+               }
+             ]
+           } = notebook
+  end
+
+  test "imports notebook with invalid vega-lite output" do
+    markdown = """
+    # My Notebook
+
+    ## Section 1
+
+    ```elixir
+    Vl.new(width: 500, height: 200)
+    |> Vl.data_from_series(in: [1, 2, 3, 4, 5], out: [1, 2, 3, 4, 5])
+    |> Vl.mark(:line)
+    |> Vl.encode_field(:x, "in", type: :quantitative)
+    |> Vl.encode_field(:y, "out", type: :quantitative)
+    ```
+
+    ```vega-lite
+    "$schema":"https://vega.github.io/schema/vega-lite/v5.json","data":{"values":[{"in":1,"out":1},{"in":2,"out":2},{"in":3,"out":3},{"in":4,"out":4},{"in":5,"out":5}]},"encoding":{"x":{"field":"in","type":"quantitative"},"y":{"field":"out","type":"quantitative"}},"height":200,"mark":"line","width":500}
+    ```
```

Comment:
Nit: I'd simplify this to keep the test focused, like source `:ok` and output `not_a_json` or whatever :)

Reply:
OK, it's done in bebdd2d